pkg/config/default_darwin.go

```diff
@@ -20,3 +20,7 @@ func getDefaultMachineVolumes() []string {
 		"/var/folders:/var/folders",
 	}
 }
+
+func getDefaultComposeProviders() []string {
+	return append(defaultUnixComposeProviders, "/opt/homebrew/bin/docker-compose", "podman-compose")
```

Comment:
I think you can add `$HOME/.docker/cli-plugins/docker-compose` for macOS as well

Reply:
That's added via [defaultUnixComposeProviders](https://github.com/containers/common/pull/1571/files/b62ac75367f5b95d2a7c41eb177d8980cf612a80#diff-f0c37908c782272aaba935b24140f7c15106c3f153c4536fa27700f3535d7567R11)

Reply:
but does it make sense to use 
```
"/usr/local/lib/docker/cli-plugins/docker-compose",
"/usr/local/libexec/docker/cli-plugins/docker-compose",
"/usr/lib/docker/cli-plugins/docker-compose",
"/usr/libexec/docker/cli-plugins/docker-compose"
```
	on macOS ?

Reply:
`/opt/homebrew/bin/docker-compose`

please note that on intel CPUs, it's not in /opt/homebrew

> https://docs.brew.sh/Installation
This script installs Homebrew to its default, supported, best prefix (/usr/local for macOS Intel, /opt/homebrew for Apple Silicon

Reply:
I think those work on Linux, so I assumed they work on Mac as well.  If you have some pointers that would be great.

Reply:
but neither docker desktop or package manager will install something in /usr/lib or /usr/libexec it's system read-only

https://support.apple.com/en-us/HT204899

and `/usr/local/lib/docker` or `/usr/local/libexec/docker` I don't think anyone install it something there

Reply:
Very valuable feedback, thanks!  Will update with the next push.

Reply:
`/Applications/Docker.app/Contents/Resources/cli-plugins/docker-compose` as well

Reply:
> `/opt/homebrew/bin/docker-compose`
> 
> please note that on intel CPUs, it's not in /opt/homebrew

`/usr/local/bin/docker-compose` should be the location for intel macs, but when installing using homebrew, homebrew automatically emits a warning asking the user to link it to `~/.docker/cli-plugins/docker-compose`. I'd use both just in case.

Reply:
Done :heavy_check_mark: 

PTanotherL